### ISSUE TITLE
3.0.1 couldn't working with xcode 8

### ISSUE THREAD

**Issue by arden** [aschuch/AwesomeCache#85]
3.0.1 couldn't working with xcode 8

**Comment by aschuch** [comment]
Yes, the project will be updated to work with Swift 2.3 and 3.0 very soon.

**Comment by aschuch** [comment]
The Swift 2.3 version is tagged as `4.0`, master contains a working Swift 3.0 version.

**Comment by alejandrorbradford** [comment]
So if I migrate my project, the framwork should work with no problems?

**Comment by aschuch** [comment]
Yes, use tag `4.0` for Swift 2.3 and the master branch for Swift 3.0

**Comment by mtorpvret** [comment]
Great! When will 4.0 be available as a cocoapod?

**Comment by ikovalisko** [comment]
Also, would like to hear when 4.0 podspec will be on CocoaPods?

**Comment by Zkriya** [comment]
Waiting for 4.0 on cocoapod

**Comment by marcbeaudoin** [comment]
Great! When will 4.0 be available as a cocoapod?

**Comment by mneuwert** [comment]
I would also like to switch to 4.0 as cocoapod. Thanks in advance! Please provide the pod spec!
